homeassistant/components/simplisafe/__init__.py

```diff
@@ -406,6 +406,7 @@ def _on_event(self, data):
         _LOGGER.debug("New websocket event: %s", event)
         self.last_events[data["sid"]] = event
         async_dispatcher_send(self._hass, TOPIC_UPDATE.format(data["sid"]))
+        self._hass.bus.async_fire(EVENT_SIMPLISAFE_EVENT, event_data=asdict(event))
```

Comment:
We shouldn't fire every event. We should only fire events that are not captured by entities: `camera_motion_detected`, `entry_detected`, `doorbell_detected`, `motion_detected`.